lib/src/extension.dart

```diff
@@ -1,21 +1,37 @@
+import 'package:sorted/src/sorting_strategies/default.dart';
+import 'package:sorted/src/sorting_strategies/merge.dart';
+import 'package:sorted/src/sorting_strategy.dart';
+
 import 'rule.dart';
 
 extension Sorted<T> on Iterable<T> {
-  /// Returns a sorted list according to the `rules`. Order matters, so if
+  /// Returns a sorted list according to the [rules]. Order matters, so if
```

Comment:
```suggestion
  /// Returns a sorted list according to the `rules`. Order matters, so if
```

Square brackets don't make valid links to method/function parameters